docs/content/configuration.md

```diff
@@ -70,17 +70,22 @@ these are rarely used for various reasons.
+| config file                   | environment                    | **default** and example value                                                             | description                                                                                                                                                               |
+| ----------------------------- | ------------------------------ | ------------------------------------------------------------------------------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `hsts`                        |                                | `{"enable": true, "maxAgeSeconds": 31536000, "includeSubdomains": true, "preload": true}` | [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) options to use with HTTPS (default is the example value, max age is a year)                          |
+|                               | `CMD_HSTS_ENABLE`              | **`true`** or `false`                                                                     | set to enable [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) if HTTPS is also enabled (default is ` true`)                                          |
+|                               | `CMD_HSTS_INCLUDE_SUBDOMAINS`  | **`true`** or `false`                                                                     | set to include subdomains in HSTS (default is `true`)                                                                                                                     |
+|                               | `CMD_HSTS_MAX_AGE`             | **`31536000`**, `60 * 60 * 24 * 365`                                                      | max duration in seconds to tell clients to keep HSTS status (default is a year)                                                                                           |
+|                               | `CMD_HSTS_PRELOAD`             | **`true`** or `false`                                                                     | whether to allow preloading of the site's HSTS status (e.g. into browsers)                                                                                                |
+| `csp`                         |                                |                                                                                           | Nested object to configure the Content Security Policy                                                                                                                    |
```

Comment:
I wouldn't remove the example here. Otherwise people might try to configure e.g. the key "csp.enable".